src/client/pages/EmailSent.tsx

```diff
@@ -63,74 +69,75 @@ export const EmailSent = ({
 			) : (
 				<MainBodyText>We’ve sent you an email.</MainBodyText>
 			)}
-			<MainBodyText>Please follow the instructions in this email.</MainBodyText>
-			<MainBodyText>
-				<b>The link is valid for 60 minutes.</b>
-			</MainBodyText>
-			{changeEmailPage && (
+			{instructionContext ? (
 				<MainBodyText>
-					Wrong email address?{' '}
-					<Link href={`${changeEmailPage}${queryString}`}>
-						Change email address
-					</Link>
-					.
+					Please follow the link in the email to {instructionContext}.
+				</MainBodyText>
+			) : (
+				<MainBodyText>
+					Please follow the instructions in this email.
 				</MainBodyText>
 			)}
-			{email && resendEmailAction && (
-				<>
-					<InfoSummary
-						message="Didn’t receive an email?"
-						context={
-							<>
-								If you can’t find the email in your inbox or spam folder, please
-								click below and we will send you a new one.
-								{noAccountInfo && (
-									<>
-										<br />
-										<b>
-											If you don’t receive an email within 2 minutes you may not
-											have an account.
-										</b>
-										<br />
-										Don’t have an account?{' '}
-										<Link href={`${buildUrl('/register')}${queryString}`}>
-											Register for free
-										</Link>
-									</>
-								)}
-							</>
-						}
-					/>
-					<MainForm
-						formAction={`${resendEmailAction}${queryString}`}
-						submitButtonText={'Resend email'}
-						submitButtonPriority="tertiary"
-						submitButtonHalfWidth
-						recaptchaSiteKey={recaptchaSiteKey}
-						setRecaptchaErrorContext={setRecaptchaErrorContext}
-						setRecaptchaErrorMessage={setRecaptchaErrorMessage}
-						formTrackingName={formTrackingName}
-						disableOnSubmit
-						formErrorMessageFromParent={formError}
-					>
-						<EmailInput defaultValue={email} hidden hideLabel />
-					</MainForm>
-				</>
-			)}
-			{showHelpBox && (
-				<MainBodyText cssOverrides={belowFormMarginTopSpacingStyle}>
-					If you are still having trouble, contact our customer service team at{' '}
-					<ExternalLink
-						cssOverrides={css`
-							font-weight: 700;
-						`}
-						href={locations.SUPPORT_EMAIL_MAILTO}
-					>
-						{SUPPORT_EMAIL}
-					</ExternalLink>
+			<MainBodyText>
+				<b>
+					For your security, the link in the email will expire in 60 minutes.
+				</b>
+			</MainBodyText>
+			<div css={helpBoxStyles}>
```

Comment:
Could this not be using `TermsBox` (which I suggested we rename)?

Reply:
Refactored and renamed into `InformationBox`